modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -16,10 +16,9 @@ class DisabilityCompensationController < ClaimsApi::V2::Veterans::Base
         FORM_NUMBER = '526'
 
         before_action :verify_access!
+        before_action :shared_validation, only: %i[submit validate]
```

Comment:
Do you think it would be preferred using symbols here for code consistency?  (at least from what I was able to find being done)  `only: [:submit, :validate]`